src/server.ts

```diff
@@ -0,0 +1,66 @@
+import { credentials, Metadata } from '@grpc/grpc-js';
+import express from 'express';
+import { Express } from 'express';
+import jayson from 'jayson';
+import NodeClient from './client';
+import getJsonRpcMethods from './methods';
+import logger from './logger';
+import { v4 as uuidv4 } from 'uuid';
+
+export default (
+    nodeAddress: string,
+    nodePort: number,
+    nodeTimeout: number
+): Express => {
+    const metadata = new Metadata();
+    metadata.add('authentication', 'rpcadmin');
+    const insecureCredentials = credentials.createInsecure();
+    const nodeClient = new NodeClient(
+        nodeAddress,
+        nodePort,
+        insecureCredentials,
+        metadata,
+        nodeTimeout
+    );
+    const server = new jayson.Server(getJsonRpcMethods(nodeClient));
+
+    const app = express();
+    app.use(express.json());
+    app.post('/json-rpc', (req, res, next) => {
```

Comment:
remove json-rpc maybe :pleading_face:

Reply:
Or make it a parameter?

Reply:
I have just removed it. It is in the interest of everyone for it to be somewhat consistent across people who run the server.